contributing.md

```diff
@@ -15,17 +15,17 @@ This file contains notes for potential contributors to GPflow, as well as some n
 
-We welcome contributions to GPflow. If you would like to contribute a feature, please raise discussion via a GitHub issue, to discuss the suitability of the feature within GPflow. If the feature is outside the envisaged scope, we can still link to a separate project in our Readme. Large features also make it onto the [roadmap](roadmap.md).
+We welcome contributions to GPflow. If you would like to contribute a feature, please raise discussion via a GitHub issue, to discuss the suitability of the feature within GPflow. If the feature is outside the envisaged scope, we can still link to a separate project in our Readme. <!-- Large features also make it onto the [roadmap](roadmap.md). -->
 
 ### I have this big feature/extension I would like to add...
 
 Due to limited scope we may not be able to review and merge every feature, however useful it may be. Particularly large contributions or changes to core code are harder to justify against the scope of the project or future development plans. For such contributions, we suggest you publish them as a separate package that extends GPflow. We can link to your project from an issue discussing the topic or within the repository. Discussing a possible contribution in an issue should give an indication to how broadly it is supported to bring it into the codebase.
 
 ### ...but it won't work without changes to GPflow core?
 
-We aim to have the GPflow core infrastructure be sufficiently extensible and modular to enable a wide range of third-party extensions without having to touch the core of GPflow. The `inducing_variables` module is an example of this to enable interdomain approximations (multiscale inducing features, Fourier features, etc.). If your feature/extension does not work outside of GPflow-core because something is hard-coded, please open an issue to discuss this with us!
+We aim to have the GPflow core infrastructure be sufficiently extensible and modular to enable a wide range of third-party extensions without having to touch the core of GPflow. The `inducing_variables` module is an example of this to enable interdomain approximations (multiscale inducing features, Fourier features, etc.). If your feature/extension does not work outside of GPflow-core because something is hard-coded, please open an issue to discuss this with us! We are happy to discuss and implement changes to the core code that make it easier for you to extend GPflow with a separate package.
```

Comment:
> We are happy to discuss and implement changes to the core code that make it easier for you to extend GPflow with a separate package.

are we definitely happy to implement the changes?

Reply:
if we discuss and agree we should do it, yes - I don't want it to seem like it'd be up to the external person to fix gpflow-core to make it more modular&extendible.

Reply:
> if we discuss and agree we should do it

exactly. but this isn't mentioned in the text (the fact that we'd need to agree to it)

Reply:
Then, please, leave a suggestion as to how you'd word it. I don't see the issue you seem to have with this change.

Reply:
```suggestion
We aim to have the GPflow core infrastructure be sufficiently extensible and modular to enable a wide range of third-party extensions without having to touch the core of GPflow. The `inducing_variables` module is an example of this to enable interdomain approximations (multiscale inducing features, Fourier features, etc.). If your feature/extension does not work outside of GPflow-core because something is hard-coded, please open an issue to discuss this with us! We are happy to discuss changes to the core code that make it easier for you to extend GPflow with a separate package.
```

Reply:
suggestion: remove "and implement"